v2.1/scalar-expressions.md

```diff
@@ -670,6 +605,71 @@ Check our blog for
 The operand must be implicitly coercible to the given type.
 The result has the given type.
 
+## Subquery Expressions
+
+### Scalar Subqueries
+
+Syntax:
+
+~~~
+( ... subquery ... )
+~~~
+
+Evaluates the subquery, asserts that it returns a single row and single column,
+and then evaluates to the value of that single cell. Any [selection query](selection-queries.html)
+can be used as subquery.
+
+For example:
+
+~~~sql
```

Comment:
Would you mind added copy-to-clipboard functionality to this and all other command examples on this page?